openmaptiles/sql.py

```diff
@@ -225,4 +250,19 @@ def field_map(match):
         sql = re.sub(
             r'/\*\s*DELAY_MATERIALIZED_VIEW_CREATION\s*\*/', ' WITH NO DATA ', sql)
 
+    exist_check = ""
```

Comment:
I think it is better to just add all this at the top of the function as a "result" var, and keep appending to it. Also, we can use a much simpler `SELECT '<table>'::regclass;`  here instead. 

```
     result = ''
     for table in layer.required_tables:
         result += f"-- Assert {table} exists\nSELECT '{table}'::regclass;\n\n"
     for func in layer.required_functions:
         result += f"-- Assert {func} exists\nSELECT '{func}'::regprocedure;\n\n"
     ...
     result += sql
     return result
```